docs/Feature Documentation/Audio and Video Streaming Guide/index.md

```diff
@@ -1,28 +1,18 @@
 ## Initial Configuration
 
-### SDL Core Setup
-
-If you don't already have Core built, you can follow the [Install and Run Guide](../../getting-started/install-and-run/).
```

Comment:
This guide was cluttered and a bit hard to follow so I removed it trying to clean up the flow. You are welcome to add it back, I just didn't think it fit in this guide.

Reply:
Eh, I think it needs to at least be mentioned, since it is a prerequisite for everything in this guide. Perhaps I will add it in as a smaller footnote